core/storage/audit/gae_models_test.py

```diff
@@ -104,6 +125,23 @@ def test_get_deletion_policy(self) -> None:
             audit_models.UsernameChangeAuditModel.get_deletion_policy(),
             base_models.DELETION_POLICY.KEEP)
 
+    def test_get_export_policy(self) -> None:
+        sample_dict = base_models.BaseModel.get_export_policy()
+        sample_dict.update(
+            {'committer_id': base_models.EXPORT_POLICY.NOT_APPLICABLE,
```

Comment:
I think @seanlip requested already that the `{` be moved to the previous line. When a reviewer says `Ditto elsewhere.` please make sure you actually fix it elsewhere

Reply:
Done!